documentation/asciidoc/stories/assembly_operator_deployments.adoc

```diff
@@ -4,7 +4,7 @@ ifdef::context[:parent-context: {context}]
 = {ispn_operator}
 
 [role="_abstract"]
-{ispn_operator} provides operational intelligence and reduces management complexity for deploying {brandname} on {k8s} and {openshift}.
+{ispn_operator} provides operational intelligence and reduces management complexity for deploying {brandname} on Kubernetes and {openshift}.
```

Comment:
Why is `{k8s} a problem? This is used throughout the docs.

Reply:
{k8s} resolves downstream as Openshift and in this sentence we don't want the substitution